.changelog/epilogue.md

```diff
@@ -0,0 +1,556 @@
+## [v10.0.1] 2023-05-25
```

Comment:
I don't understand the purpose of this doc, why it is so long and why does it contain changes from a while ago?

The unclog doc states:
```
|__ epilogue.md               - Any content to be added to the end of the generated CHANGELOG.
```

But main does not really have a changelog because only `release/v*` branches are used to cut releases?

Reply:
Running `unclog build > CHANGELOG.md` will build the changelog with info from `.changelog`. As we'll not add all the historical changelog to this folder, we just add the current content of CHANGELOG.md to epilogue.md. See https://github.com/informalsystems/unclog#initializing-a-changelog for more details.